web_ir_actions_act_window_page/README.md

```diff
@@ -0,0 +1,12 @@
+Client side paging
+=====================
+
+This addon enables buttons to return::
```

Comment:
Don't know. We use it for buttons, when performing some action + move to next record.